suites/squid/rgw/tier-1_rgw_archive_ms_upgrade_71GA_to_8x_latest.yaml

```diff
@@ -377,17 +377,6 @@ tests:
             command: start
             service: upgrade
             verify_cluster_health: true
-  - test:
-      name: Verify DBR feature enabled on upgraded cluster
```

Comment:
Looks like this test case present for this upgrade path, in only one place .
Can you please recheck.

Reply:
its present in : squid/rgw/tier-1_rgw_ms_upgrade_61GA_to_8x_latest.yaml (in pre upgrade)and squid/rgw/tier-2_rgw_ms_regression_test.yaml
but will retrieve it as its needed in brownfield environment